### PR TITLE
docs: Update Shiny example

### PR DESCRIPTION
**Pull request by gadenbuie**
Makes the Shiny example runnable.

Also, because we're in chatlas, I'd prefer calling the actual chat model `chat` and separately calling the `ui.Chat()` instance `chat_ui()`. I think this naming makes it clearer which things are interacting (chatting) with the model (`chat`) and which things are just UI in the app `ui.Chat()`.

### CODE REVIEW

File: docs/web-apps.qmd [hunk 1]
```diff
@@ -11,23 +11,25 @@ This is a great way to quickly test your model, but you'll likely want to embed
 
 ## Shiny
 
-Pass the result of `.chat()` directly to Shiny's [`ui.Chat` component](https://shiny.posit.co/py/components/display-messages/chat/) to create a chat interface in your own [Shiny](https://shiny.rstudio.com/py) app.
+Pass the result of `chat.stream()` directly to Shiny's [`ui.Chat` component](https://shiny.posit.co/py/components/display-messages/chat/) to create a chat interface in your own [Shiny](https://shiny.rstudio.com/py) app.
```
Comment (gadenbuie) [inline docs/web-apps.qmd]: On second read, I don't think I got this right and I'm not quite sure what you were wanting to say here. I think maybe the example is out of sync with the text.

Comment (cpsievert) [reply docs/web-apps.qmd]: Ah, I mean to say to pass it to `.append_message_stream()`

Comment (cpsievert) [inline docs/web-apps.qmd]:
```suggestion
Using Shiny's [`ui.Chat` component](https://shiny.posit.co/py/components/display-messages/chat/), you can simply pass user input from the component into the `chat.stream()` method. This generate a response stream that can then be passed to `.append_message_stream()`.
```

**Comment by cpsievert** [comment]
Thanks!

**Pull request merged**